packages/api/lambdas/sqs-message-consumer.js

```diff
@@ -42,8 +48,35 @@ async function processQueues(event, dispatchFn) {
  *
  */
 function dispatch(message) {
-  const input = Object.assign({}, message.Body);
-  return rulesHelpers.queueMessageForRule(this.rule, input);
+  const queueUrl = this.rule.rule.value;
+  const messageReceiveCount = parseInt(message.Attributes.ApproximateReceiveCount, 10);
+
+  if (get(this.rule, 'meta.retries', 3) < messageReceiveCount - 1) {
+    log.debug(`message ${message.MessageId} from queue ${queueUrl} has been processed ${messageReceiveCount - 1} times, no more retries`);
```

Comment:
What is this doing here? We're saying no more retries but putting it back on the queue? Should this go to a DLQ?

Reply:
The message is ignored if the retries limit is reached.  The message will be removed to DLQ by the SQS service based on the RedrivePolicy for the DLQ.

Reply:
Ok thanks! Tested it out and it works!